lifecycle/index.d.ts

```diff
@@ -0,0 +1,35 @@
+import type { WritableStore, ReadableStore } from '../atom/index.js'
+
+type Store<Data> = WritableStore<Data> | ReadableStore<Data>
+
+export function onSet<Data>(
+  store: Store<Data>,
+  handler: (payload: {
```

Comment:
What do you think about removing `payload` key? For the reason of shortness and byte saving =^_^=